test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -41,6 +44,11 @@
 (deftest ^:integration old-master-route-config
   (testing "The old map-style route configuration map still works."
     (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "MOCK.TODO: discuss - this is *probably* ok because all we really care about
+     is that the webserver is accepting connections and handing them to the correct
+     ring handler.  Would be better if we could implement this to hit a clojure
+     endpoint instead of a Ruby one but I don't remember if there are any pure-clojure
+     endpoints under /puppet."
```

Comment:
:fried_shrimp: :books: doesn't matter much to me if you want to take it that far.  I probably wouldn't do that until we had a use case, doesn't seem like it would be hard to  add in later?  But if we could just plumb in a fn for 'request-handler', then several of the tests on our current list I think I'd be satisfied with just passing in a 200 handler and keeping the mocking turned on.